apps/website/src/pages/docs/listbox.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Listbox
+description: TODO Write Listbox description
```

Comment:
There is still some TODOs for the documentation of this component. I'm not exactly sure how to describe this component without repeating the `List` documentation

Reply:
Maybe use something like "The Listbox component is used to display a `ListItem`s within the confines of a box"? I would just sprinkle "in a box" around haha